File: src/pages/exercises/function-area-of-circle/_codes/ruby/code/spec/area_of_circle.spec.rb
require_relative "../src/area_of_circle"

describe AreaOfCircle do
    describe ".circle" do
        context "Area of circle with radius of 2" do
            it "return 12.56" do
                expect(AreaOfCircle.compute(2)).to eql(12.56)
            end
        end

        context "Area of circle with radius of 3" do
            it "return 28.26" do
                expect(AreaOfCircle.compute(3)).to eql(28.26)
            end
        end
    end
end

File: src/pages/exercises/function-area-of-circle/_codes/php/code/spec/area-of-circle.spec.php
<?php

require __DIR__ . '/../src/area-of-circle.php';

describe('Circle Tools', function () {
  it('Area of the circle of radius 1 is of 3.1415926535898', function () {
    // https://stackoverflow.com/questions/3148937/compare-floats-in-php
    expect(round(areaOfCircle(1), 3))->toEqual(round(3.1415926535898, 3));
  });

  xit('Area of the circle of radius 10 is of 314.15926535898', function () {
    expect(round(areaOfCircle(10), 3))->toEqual(round(314.15926535898, 3));
  });
});
 